packages/server/src/database.test.ts

```diff
@@ -49,6 +50,9 @@ describe('Database config', () => {
           if (sql === 'SELECT pg_try_advisory_lock($1)') {
             result.rows = [{ pg_try_advisory_lock: advisoryLockResponse } as unknown as R];
           }
+          if (sql === mockQueries.GetVersionSql) {
+            result.rows = [{ version: preDeployVersion } as unknown as R];
```

Comment:
@mattlong  - not sure if you know why this was necessary?  or how it was working before?  I was getting test failures without it.

I think this is the first pre-deploy migration that uses `idempotentCreateIndex`, which has more complex logic than we were handling with this mock postgres client.

By adding this check, we essentially trick the setup code into thinking that all pre-deploy migrations are already run.